### PR TITLE
Rename files to be valid python identifiers

### PR DESCRIPTION
**Pull request by jaedolph**
As identified here:  #198

### CODE REVIEW

File: tests/get_total_coverage.sh [hunk 1]
```diff
@@ -20,13 +20,13 @@ tox -e py26,py27,py36,py37 -- --cov-append
```
Comment (jaedolph) [inline tests/get_total_coverage.sh:20]: Couldn't find this file referenced anywhere. Do you know where it gets called from?

Comment (tyll) [reply tests/get_total_coverage.sh:20]: This is only called manually. It needs to be integrated into a CI or test-harness needs to be changed to call `get_coverage.sh` or something else to create coverage data. But nothing happened in this direction, yet.

Comment (jaedolph) [reply tests/get_total_coverage.sh:20]: Ah right, should be ok to change the name then? I have changed all the references to the other coverage scripts it references (e.g. `get_coverage.sh`)

**Comment by jaedolph** [comment]
[citest bad]

File: tests/get_total_coverage.sh (hunk 1 above)
Comment (tyll) [reply tests/get_total_coverage.sh:20]: Probably not needed but it does not hurt, too.

**Comment by jaedolph** [comment]
[citest bad]

**Comment by tyll** [comment]
FYI: This has a conflict that needs to be resolved

**Comment by jaedolph** [comment]
> FYI: This has a conflict that needs to be resolved

Fixed now

**Comment by jaedolph** [comment]
[citest bad]

**Pull request merged**